lib/git.ml

```diff
@@ -164,3 +164,10 @@ let tag tag_name tag_action =
 
 let uncommit () = Process.proc "git reset HEAD~1"
 let unstash () = Process.proc "git stash pop"
+
+let done_ () =
+  let prev_branch = get_current_branch () in
+  let main_branch = fetch_main_branch () in
+  switch None;
```

Comment:
I think this can be optimised a little bit, and we can pass already fetched branch
```suggestion
  switch (Some main_branch);
```